deployment/network-operator/README.md

```diff
@@ -0,0 +1,163 @@
+# Nvidia Mellanox Network Operator Helm Chart
+
+Nvidia Mellanox Network Operator Helm Chart provides an easy way to install, configure and manage
+the lifecycle of Nvidia Mellanox network operator.
+
+## Nvidia Mellanox Network Operator
+Nvidia Mellanox Network Operator leverages [Kubernetes CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+and [Operator SDK](https://github.com/operator-framework/operator-sdk) to manage Networking related Components in order to enable Fast networking, 
+RDMA and GPUDirect for workloads in a Kubernetes cluster.
+
+The Goal of Network Operator is to manage _all_ networking related components to enable execution of
+RDMA and GPUDirect RDMA workloads in a kubernetes cluster including:
+* Mellanox Networking drivers to enable advanced features
+* Kubernetes device plugins to provide hardware resources for fast network
+* Kubernetes secondary network for Network intensive workloads
+
+## QuickStart
+
+### Prerequisites
+
+- Kubernetes v1.15+
+- Helm v3
+- Ubuntu 18.04LTS
+
+
+### Install Helm
+
+Helm provides an install script to copy helm binary to your system:
+```
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
+```
+
+For additional information and methods for installing Helm, refer to the official [helm website](https://helm.sh/)
+
+### Deploy Network Operator
+
+```
+# Add Repo
+$ helm repo add mellanox https://mellanox.github.io/network-operator
+$ helm repo update
+
+# Install Operator
+$ helm install -n network-operator --create-namespace --wait mellanox/network-operator network-operator
+
+# View deployed resources
+$ kubectl -n network-operator get pods
+```
+
+>__Note:__ By default the operator is deployed without an instance of `NicClusterPolicy`
+custom resource. The user is required to create it later with configuration matching the cluster or use
+chart parameters to deploy it together with the operator.
+
+## Chart parameters
+
+In order to tailor the deployment of the network operator to your cluster needs
+We have introduced the following Chart parameters.
+
+### General parameters
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `nfd.enabled` | bool | `True` | deploy Node Feature Discovery |
+| `operator.repository` | string | `mellanox` | Network Operator image repository |
+| `operator.image` | string | `network-operator` | Network Operator image name |
+| `operator.tag` | string | `None` | Network Operator image tag |
```

Comment:
do we have default for tag?

Reply:
no default for it, if it is not defined then the image tag is the chart appVersion